### ISSUE TITLE
paginator: mat-select for items per page is not accessible when focusing on items 

### ISSUE THREAD

**Issue by danw5050** [angular/components#10726]
### Bug, feature request, or proposal:
The mat-select which is used within the datatable to select the number of items in a page is not read by screen readers (ChromeVox and VoiceOver) as the user changes the number which is focus on the extended dropdown. It is also does not read the items per page on the initial focus of the datatable.

### What is the expected behavior?
The  screenreader to be able to pick up on the current number (representing items/page within datatable) that is in focus. It should also say the items per page of the datatable on initial focus of the element.

### What is the current behavior?
The screenreader does not pick up on the number (representing items/page within datatable) that is in focus.

### What is the use-case or motivation for changing an existing behavior?
To add better accessibility for the datatable

### Which versions of AngularJS, Material, OS, and browsers are affected?
Chrome, Mac

**Comment by crisbeto** [comment]
Related to https://github.com/angular/material2/issues/11083.

**Comment by MrPerfect8699** [comment]
Do we have any solution yet on this issue? I don't want the alternative, just need to fix from angular team. We have to fix manually all the mat-select(that we can do) but cannot fix accessibility issue in mat-pagination since the select box in inbuilt

**Comment by jelbourn** [comment]
This should have been fixed by #20082
